Sources/PackageDescription/PackageDescriptionSerialization.swift

```diff
@@ -201,7 +201,7 @@ enum Serialization {
 
     struct Target: Codable {
         let name: String
-        let group: TargetGroup
+        let group: TargetGroup?
```

Comment:
This was needed to make `testSimpleAPI` work, I'm not 100% sure why we're seeing it more widespread.